src/controllers/v1/profile.js

```diff
@@ -97,6 +97,36 @@ module.exports = class Mentees {
 		}
 	}
 
+	/**
+	 * Get mentor or mentee extension by user ID.
```

Comment:
change function description @sumanvpacewisdom

Reply:
i have done the changes and it is resolved.